common/src/main/scala/hydra/common/util/InstantUtils.scala

```diff
@@ -0,0 +1,14 @@
+package hydra.common.util
+
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, LocalDate, ZoneId}
+
+object InstantUtils {
```

Comment:
actually we need it only in one place - in config decoders. So for now separate util class is redundant. We can move it to configs and add one more test for it